### ISSUE TITLE
is chocolat project dead?

### ISSUE THREAD

**Issue by miltar** [atg/chocolat-public#1823]
Just this.

**Comment by karlsander** [comment]
I REALLY hope not. It never updated to often, so I wouldn't generally be worried, but the Help link being offline for so long is a little worrying. @atg?

There is no mac native text editor alternative on its level. I'm personally even down for yet another paid update if its not financially viable otherwise.

**Comment by wkoszek** [comment]
@SilverTab ^^

**Comment by gobijan** [comment]
I'm also checking on updates quite regularly but still no luck. Chocolat offers a nice native integration that vscode, sublime, atom and all the other cross platform ones are lacking.
Also the fact that it has built in quite a nice intellisense is definitely a plus point. I'd also be up for a 4.0 paid upgrade and I guess many others too. 

When it comes to native there are not so many options:
Textmate (slow development and not offering any intellisense)
CodeRunner (quite good all-round intellisense and built in debugger but lacking recent development)
BBEdit (solid product but just feels a little off to me)

**Comment by miltar** [comment]
I agree and it would really be a shame if the project did not continue, but I think that silence says everything.

**Comment by chikega** [comment]
C'est triste Chocolat ...

**Comment by tatecapone** [comment]
definitely textmate is the king again

**Comment by amcgregor** [comment]
There's always [Vim](https://github.com/amcgregor/dotfiles/).

![I don't often run it like this, though.](https://ghcdn.webcore.io/vim/full-ide-experience.png)
